src/WebJobs.Script.WebHost/App_Data/secrets/.azurefunctions.swagger.json

```diff
@@ -0,0 +1,9 @@
+﻿{
+  "keys": [
```

Comment:
I think this plan of creating this special function key is hacky. I've got a better idea that we should discuss.